ui/app/pages/settings/contact-list-tab/view-contact/view-contact.component.js

```diff
@@ -48,28 +66,41 @@ export default class ViewContact extends PureComponent {
           </div>
           <div className="address-book__view-contact__group">
             <div className="address-book__view-contact__group__label">
-              { t('ethereumPublicAddress') }
+              {t('ethereumPublicAddress')}
             </div>
             <div className="address-book__view-contact__group__value">
-              <div
-                className="address-book__view-contact__group__static-address"
-              >
-                { quadSplit(checkSummedAddress) }
+              <div className="address-book__view-contact__group__static-address">
+                {quadSplit(checkSummedAddress)}
               </div>
-              <button
-                className="address-book__view-contact__group__static-address--copy-icon"
-                onClick={() => copyToClipboard(checkSummedAddress)}
+              <Tooltip
+                wrapperClassName="address-book__tooltip-wrapper"
```

Comment:
It looks like this class doesn't exist? :thinking: If so, better to not reference it.
```suggestion
```